scheduler/filter/filter.go

```diff
@@ -14,6 +15,9 @@ type Filter interface {
 
 	// Return a subset of nodes that were accepted by the filtering policy.
 	Filter(*cluster.ContainerConfig, []*node.Node, bool) ([]*node.Node, error)
+
+	// Return a list of constraints/filters provided
+	GetAllFilters(*cluster.ContainerConfig) ([]string, error)
```

Comment:
Alright, I'll make the change. Any particular reason for preferring one over the other?